website/src/docs/webcam.md

```diff
@@ -156,6 +157,12 @@ Set the preferred mime type for images, for example `'image/png'`. If the browse
 
 If no preferred image mime type is given, the Webcam plugin will prefer types listed in the [`allowedFileTypes` restriction](/docs/uppy/#restrictions), if any.
 
+### `mobileNativeCamera`
+
+Replaces Uppy’s custom camera UI on mobile with “Take Picture” and / or “Record Video” buttons (depending on [`modes`](#modes) option) that open native device UI for pictures / video (`Function: boolean` || `boolean`, default: `isMobile()`).
+
+Set to a boolean to forcefully enable / disable this feature, or a function which returns a boolean. By default we use the [`is-mobile`](https://github.com/juliangruber/is-mobile) package.
```

Comment:
Some changes to better align the new docs (one sentence + type, clarifications below) and improved grammar. 
```suggestion
Replaces Uppy’s custom camera UI on mobile with the native device camera (`Function: boolean` || `boolean`, default: `isMobile()`).

This will show the “Take Picture” and / or “Record Video” buttons, which ones show depends on the [`modes`](#modes) option.

You can set a boolean to forcefully enable / disable this feature, or a function which returns a boolean. By default we use the [`is-mobile`](https://github.com/juliangruber/is-mobile) package.
```